_distro_map.yml

```diff
@@ -173,12 +179,9 @@ openshift-rosa:
   site_name: Documentation
   site_url: https://docs.openshift.com/
   branches:
-    enterprise-4.12:
-      name: ''
-      dir: rosa/
     enterprise-4.13:
       name: ''
-      dir: rosa-hcp
+      dir: rosa/
```

Comment:
@EricPonvelle, at GA, I'm going to switch all of ROSA over to 4.13. This will remove the separate build for the ROSA HCP preview. Last we talked, this seemed fine. If something's changed, please adjust the topic map so that what you want to go live tomorrow will show up and tag me in the PR so I can grant it an exception.

Reply:
There are no changes that I am aware of, and I think we can proceed as planned! Thanks for your help!

Reply:
I am glad to offer it! Thank you for confirming. :)